sql/parser/eval.go

```diff
@@ -396,6 +398,21 @@ var cmpOps = map[cmpArgs]func(Datum, Datum) (DBool, error){
 	cmpArgs{LE, intervalType, intervalType}: func(left Datum, right Datum) (DBool, error) {
 		return DBool(left.(DInterval).Duration <= right.(DInterval).Duration), nil
 	},
+
+	cmpArgs{Like, stringType, stringType}: func(left Datum, right Datum) (DBool, error) {
+		pattern := regexp.QuoteMeta(string(right.(DString)))
+		pattern = strings.Replace(pattern, "%", ".*", -1)
+		pattern = strings.Replace(pattern, "_", ".", -1)
+		pattern = fmt.Sprintf("^%s$", pattern)
+		fmt.Println(pattern)
+
+		re, err := regexp.Compile(pattern)
```

Comment:
We'll have to figure out how to cache the compiled regex for SIMILAR TO anyways. I don't have a good thought for how to do this now, so let's just add a TODO and/or file an issue.